src/Typeahead.vue

```diff
@@ -69,12 +61,10 @@ export default {
     },
     matchCase: {
       type: Boolean,
-      coerce: coerce.boolean,
```

Comment:
Since this is removed, will need a computed property for `matchCase`. Some parts of the code will need to use the computed property when reading the variable.

Reply:
but matchCase is unused